lib/launcher.rb

```diff
@@ -46,7 +46,7 @@ def https_app
         # This is required to disable SSLv3 on Ruby 1.8.7
         ssl_options |= OpenSSL::SSL::OP_NO_SSLv2 if defined?(OpenSSL::SSL::OP_NO_SSLv2)
         ssl_options |= OpenSSL::SSL::OP_NO_SSLv3 if defined?(OpenSSL::SSL::OP_NO_SSLv3)
-        ssl_options |= OpenSSL::SSL::OP_NO_TLSv1 if defined?(OpenSSL::SSL::OP_NO_TLSv1)
+        ssl_options |= OpenSSL::SSL::OP_NO_TLSv1 if (!SETTINGS.enable_tls_v1 && defined?(OpenSSL::SSL::OP_NO_TLSv1))
```

Comment:
Could you remove the parenthesis around the condition? Rubocop doesn't like them.